### DISCUSSION TITLE
SCCM wim file builder

### DISCUSSION THREAD

**Discussion by kfriedberg** [ipxe/ipxe#920] [Show and tell]
A few years ago I made a script to do the work of converting an SCCM ISO to a WIM file for use with wimboot.  After finally cleaning it up enough to put on Github, I was hoping other people might want to give it a try to see if it's stable enough to submit to the official documentation.

https://github.com/kfriedberg/sccm-ipxe-iso-wim-converter

**Comment by mcb30** [comment]
Looks good!  :slightly_smiling_face:  I don't have an SCCM setup to test with right now: still waiting for Microsoft to resolve problems with my Visual Studio Enterprise subscription.
